lib/RoadizRozierBundle/templates/security/login.html.twig

```diff
@@ -20,46 +20,55 @@
-                       class="form-control"
-                       placeholder="{% trans %}password{% endtrans %}"
-                       required
-                       autocomplete="current-password">
+            <div class="rz-form-field rz-form-field--type-password rz-form-field--required">
+                <div class="rz-form-field__head">
+                    <label class="rz-form-field__head__label" for="_password">{% trans %}password{% endtrans %}</label>
+                </div>
+                <input
+                    is="rz-input"
+                    name="password"
+                    id="_password"
+                    type="password"
+                    autocomplete="current-password"
+                    placeholder="{% trans %}password{% endtrans %}"
+                    class="rz-form-field__input"
+                    required>
+            </div>
+            <div class="rz-form-field rz-form-field--type-checkbox">
+                <div class="rz-form-field__head">
+                    <label class="rz-form-field__head__label" for="_remember_me">{% trans %}keep_me_logged_in{% endtrans %}</label>
+                </div>
+                <input type="checkbox" class="rz-switch rz-form-field__input" id="_remember_me" name="_remember_me" checked>
             </div>
-            <div class="uk-form-row form-col-checkbox form-col--remember-me">
-                <label for="_remember_me" class="uk-form-label"><span
-                            class="uk-form-label-title">{% trans %}keep_me_logged_in{% endtrans %}</span></label><input
-                        class="rz-boolean-checkbox"
-                        type="checkbox"
-                        data-on-text="I"
-                        data-off-text="O"
-                        id="_remember_me" name="_remember_me" checked value="1"/></div>
         </div>
         <input type="hidden" name="_csrf_token" value="{{ csrf_token('authenticate') }}">
-        <fieldset>
-            <button class="uk-button uk-button-primary"
-                    type="submit"><i class="uk-icon-sign-in"></i> {% trans %}login{% endtrans %}</button>
-            <a class="uk-button uk-button-secondary"
-               href="{{ path('roadiz_rozier_login_link') }}"><i class="uk-icon-chain"></i> {% trans %}request_login_link{% endtrans %}</a>
-        </fieldset>
+        <div class="rz-button-group rz-button-group--gap-md">
+            <button type="submit" class="rz-button rz-button--primary">
+                <span class="rz-button__label">{% trans %}login{% endtrans %}</span>
+                <span class="rz-button__icon rz-icon-ri--arrow-right-s-line"></span>
+            </button>
+            <a class="rz-button rz-button--secondary" href="{{ path('roadiz_rozier_login_link') }}">
+                <span class="rz-button__label">{% trans %}request_login_link{% endtrans %}</span>
+                <span class="rz-button__icon rz-icon-ri--link"></span>
+            </a>
+        </div>
```

Comment:
@eliot488995568 This is not the final layout for buttons. `Login` should not be grouped with other actions.